### PR TITLE
Typography component

### PR DESCRIPTION
**Pull request by mehulsatardekar**
Added Typography Component

Live Preview:  https://framer-ui.netlify.app/docs/typography.html

### CODE REVIEW

File: components/Typography/typography.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
+/* @import url('../../components/root.css');
+@import url('../../components/card/card.css'); */
```
Comment (parvez-rashid) [inline components/Typography/typography.css]: Remove comments if not needed, before you merge

File: components/Typography/typography.css [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+
+@import url('../../components/framer.css');
+
+
+
```
Comment (parvez-rashid) [inline components/Typography/typography.css]: Remove extra space

File: docs/typography.html [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+                    <li class="sidebar-text flex flex-align-item-center gap-1">
+                        <box-icon name='font-family' color='#eeeded' ></box-icon>
+                        <span>Typography</span>
+                    </li>
+                   </a>
+                </ul>
+                    
+             </div>
+        </div>
+         <div class="flex mt-1 ml-1 flex-align-item-center">
```
Comment (parvez-rashid) [inline docs/typography.html:131]: I don't understand the variables `mt-1` and `ml-1` used here.

**Review by parvez-rashid — APPROVED** [review]
UI Review: UI looks great, I like the hamburger menu. However, in typography, you have to mention different sections for headings, font-size, font-weight, etc.

File: docs/typography.html (hunk 3 above)
Comment (mehulsatardekar) [reply docs/typography.html:131]: > I don't understand the variables `mt-1` and `ml-1` used here.

just a naming conventions mt means margin-top same applies to ml margin-left

File: components/Typography/typography.css [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+}
+
+.font-sm-md{
+    font-size: 1.5rem;
+}
+.font-strike{
+ text-decoration: line-through;
+}
+
+
```
Comment (j24m) [inline components/Typography/typography.css:84]: extra spacing.

File: docs/typography.html [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Typography</title>
+    <link rel="stylesheet" href="../components/Typography/typography.css">
+    <meta name="theme-color" content="#2C2929">
+
+</head>
+
+<body>
+
+
```
Comment (j24m) [inline docs/typography.html]: extra spacing.

File: docs/typography.html [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+                      <div class="ui-card-default py width-full">
+                        <div class="ui-card-text card-mb-1 flex flex-justify-center">
+                            <p class="font-bold text-gray">This is centered Text</p>
+                        </div>
+                    </div>
+
+                </div>
+
+
```
Comment (j24m) [inline docs/typography.html]: extra spacing.

**Review by j24m — APPROVED** [review]
Left some comments please check, good work.

**Pull request merged**